www/index.php

```diff
@@ -12,7 +12,7 @@
     <li><a href="/client/">Client&nbsp;Features</a></li>
     <li><a href="/map/">Map&nbsp;Features</a></li>
     <li><a href="/howto/">How&nbsp;to&nbsp;Map</a></li>
-    <li><a href="/rules/">Mapping&nbsp;Rules</a></li>
+    <li><a href="/rules/">Mapping&nbsp;Rules</a> / <a href="/guidelines/">Guidelines</a></li>
```

Comment:
It seems like the all sub pages define their own menus, so you will need to update those as well. Just do a find and replace.

Reply:
This menu is only visible on the main page.

Reply:
Ah, no nevermind. Will fix